src/server/controllers/init.js

```diff
@@ -0,0 +1,34 @@
+const axios = require('axios');
+
+const getISBN = (industryIdentifiers) => {
+    let isbn;
+    
+    industryIdentifiers.map(indId => {
+        if(indId.type === 'ISBN_13'){
+            isbn = indId.identifier;
+        }
+    });
+
+    return isbn;
```

Comment:
It looks like `find` may be better suited to this task since you're trying to find one item in an array (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find)

```js
const industryId = industryIdentifiers.find(indId => indId.type === 'ISBN_13');

return industryId?.identifier;
```

For some reading on what the `?` is there for, check out https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining